### PR TITLE
Fix path seperator, remove extra imports

### PR DESCRIPTION
**Pull request by HTFWitt**
Made UNC separator fixed character and not OS dependant.
Remove extraneous imports.

### CODE REVIEW

File: scripts/read_from_s3.py [hunk 1]
```diff
@@ -1,42 +1,36 @@
...
 import datetime
-import fsspec
-import xarray as xr
 import time
 
 import sys
+
 sys.path.append("../src")
-from UFS2ARCO import FV3Dataset
+from UFS2ARCO.fv3dataset import FV3Dataset
```
Comment (timothyas) [inline scripts/read_from_s3.py]: Is this necessary? I hope it isn't since FV3Dataset is import in src/UFS2ARCO/__init__.py

**Review by timothyas — CHANGES_REQUESTED** [review]
Looks good to me, although there are some conflicts to be resolved. Could you take care of those? Then it's good to merge

File: scripts/read_from_s3.py (hunk 1 above)
Comment (HTFWitt) [reply scripts/read_from_s3.py]: No, it's not.  
What will be an issue in the future [when we can pip install UFS2ARCO] is the use of the sys.path appendage which implies you have the source code AND that you are running the script from a location a step away from the src folder.

**Comment by timothyas** [comment]
Now it looks like the only thing that changed are the import statements, is that right?

**Comment by HTFWitt** [comment]
The last commit was to resolve the package class reference.  The extra import removal happened at the beginning of this

**Review by timothyas — APPROVED** [review]

**Pull request merged**